### PR TITLE
inital setup for action

### PR DESCRIPTION
**Pull request by claire-labry**
This is the initial setup for the action to allow us to be able to download/setup artifactory-repo-tool instead of using a script to download the tool.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+runs:
+  using: composite
+  steps:
+    - name: Install artifactory-repo-tool tool
+      id: install
+      shell: bash
+      env:
+        GH_TOKEN: ${{ inputs.github-token }}
+      run: DEST_DIR="${HOME}/bin" "${GITHUB_ACTION_PATH}/scripts/setup-artifactory-repo-tool.sh" "${{ inputs.version }}"
```
Comment (shore) [inline action.yml]: Was this supposed to use the `install-dir` input?
```suggestion
      run: DEST_DIR="${{ inputs.install-dir }}" "${GITHUB_ACTION_PATH}/scripts/setup-artifactory-repo-tool.sh" "${{ inputs.version }}"
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            required: true
+          version:
+            type: string
+            description: "Version of artifactory-repo-tool to install"
+    
+      pull_request:
+      push:
+        branches:
+          - main
+          - 'releases/*'
```
Comment (shore) [inline .github/workflows/test.yml]: This doesn't match any current branches.  Did you mean `release/*`?
```suggestion
          - 'release/*'
```

**Review by shore — CHANGES_REQUESTED** [review]
I think there should be a readme with the usual goodies -- usage, the version of ART to install by default, release process, etc.

**Comment by claire-labry** [comment]
> I think there should be a readme with the usual goodies -- usage, the version of ART to install by default, release process, etc.

💯

**Review by shore — APPROVED** [review]
LGTM!

**Pull request merged**